scripts/build_node_package.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+MODULE_PATH=/tmp/module
+BUILD_TARGET_DIR=/tmp/module_build
+
+git clone https://github.com/mmarchini-oss/node-linux-perf.git $MODULE_PATH
```

Comment:
Pin the hash by running `git reset --hard ..` to the revision that you expect (can be the latest `master`)